demo/src/app/header/header.component.ts

```diff
@@ -7,22 +7,19 @@ import { Component, OnInit } from '@angular/core';
 })
 export class HeaderComponent implements OnInit {
 
-  ngShareButtonLogo = prefixRepo('../../assets/img/logo.svg');
-  fbLogo = prefixRepo('../../assets/img/share/facebook.svg');
-  twttLogo = prefixRepo('../../assets/img/share/twitter.svg');
-  pintLogo = prefixRepo('../../assets/img/share/pinterest.svg');
-  stumbleLogo = prefixRepo('../../assets/img/share/stumbleupon.svg');
-  googleLogo = prefixRepo('../../assets/img/share/google-plus.svg');
-  tumblrLogo = prefixRepo('../../assets/img/share/tumblr.svg');
-  redditLogo = prefixRepo('../../assets/img/share/reddit.svg');
-  inLogo = prefixRepo('../../assets/img/share/linkedin.svg');
+  ngShareButtonLogo = '../../assets/img/logo.svg';
+  fbLogo = '../../assets/img/share/facebook.svg';
+  twttLogo = '../../assets/img/share/twitter.svg';
+  pintLogo = '../../assets/img/share/pinterest.svg';
+  stumbleLogo = '../../assets/img/share/stumbleupon.svg';
+  googleLogo = '../../assets/img/share/google-plus.svg';
+  tumblrLogo = '../../assets/img/share/tumblr.svg';
+  redditLogo = '../../assets/img/share/reddit.svg';
+  inLogo = '../../assets/img/share/linkedin.svg';
 
   constructor() { }
 
   ngOnInit() {
   }
 
 }
-var prefixRepo = (path) => {
-  return  'ng2-sharebuttons' + path;
-};
```

Comment:
@tinesoft The `prefixUrl` function wasn't intended for development, I disable it on `ng serve`, and only use it when deploying to `gh-pages`, assets relative paths cause 404, because it ignores the baseURL, and `prefixURL` was a workaround